docs/static_landing_page/css/landing_page_style.css

```diff
@@ -149,7 +149,7 @@ li {
 }
 .navbar li {
 	font-family: 'Code Pro Medium', Arial, sans-serif;
-	font-size: 18px;
+	font-size: 16px;
```

Comment:
What does it look like if you don't make this change? I'm trying to understand the reasoning behind this. Thanks!!!

Reply:
This is at `18px`:

![image](https://user-images.githubusercontent.com/50991099/156886575-d69737e0-bbb5-40e9-a55a-9b9a9ff35ef5.png)

`16px`: [Link](https://4917-133444044-gh.circle-artifacts.com/0/docs/_build/html/index.html)

![image](https://user-images.githubusercontent.com/50991099/156886619-4c4f3a08-1d27-4de0-9924-2ba6d8d1f1b3.png)

Reply:
The difference does not come through well in the images. You could use the inspector to change the font size and view in the browser itself using the Circle CI link.

Reply:
Is the purpose of the change to create a gap between the text items and the buttons?

Reply:
Yes 👍

Reply:
I suppose I agree with @MiroDudik in that we'd need someone with UI expertise to tell us if that's a good solution. @nessamilan perhaps?

Reply:
Thanks for the tag! 

**A few thoughts:**
- In practice, font-size changes should be made as a global change not for one instance, so it maintains a consistent type-size system throughout the site.
- I wouldn't recommend 2 buttons in the top nav. For UI, it's recommended to have a max 5-7 text links in a top navigation. Buttons are reserved for Login or one 3rd party partner (in Fairlearn's case Github as the primary creation/community platform). For example, where both Github and Login exist, it's common to see "Github" as a text link and "Login" as a button.

**Recommendation:**
- Update all 18px fonts to 16px (@iofall which looks really good in your screenshot, btw!).
- Use and icon for each Github and Discord with no label or button background. 
_Note: this subsequently makes room for a 3rd icon in the future. I'd cap these at three, more communities/social entities can be featured in the footer._
- Add tooltips for each icon.
- The Discord color looks really cool. If you decide on using the color, I'd update Github as well so they are both their brand colors. Based on the inspect tool, it looks like Github's header color is #24292f.

Reply:
Thanks @nessamilan !
The only thing I'd add is that we already use the colors in the generated sphinx header, but only on hover. By default the icons are gray or black I think. You can find the colors there to make sure they match, @iofall 🙂

Reply:
Thank you for the feedback @nessamilan. I think for this issue, I will modify both the logos (GitHub and Discord) and make them as you have suggested.

For the font size, I see that there are other design issues you have opened. I could help with those. Modifying the font size would be better tackled in those. 

> * The Discord color looks really cool. If you decide on using the color, I'd update Github as well so they are both their brand colors.

Yes, it is the official color for Discord. For GitHub, it would just be plain black and white.

Reply:
This looks excellent, @iofall !! Thanks for doing this work.  Sounds good about the font sizes in another issue. I'll go back over the list of design issues and see where it fits. You're help would be greatly appreciated!